plugins/Kaleidoscope-Qukeys/src/kaleidoscope/plugin/Qukeys.h

```diff
@@ -234,6 +247,12 @@ bool isModifierKey(Key key);
 
 extern kaleidoscope::plugin::Qukeys Qukeys;
 
+constexpr Key Qkey(Key tap_key, Key qkey) {
+  uint8_t qkey_index = Qukeys.storeQkey(qkey);
+  // TODO(EvyBongers): store the qkey index somewhere, somehow
+  return Key(kaleidoscope::ranges::QK_FIRST + tap_key.getRaw());
+}
```

Comment:
My first thought was to use the same approach as with the ModTap and LayerTap keys, but if I grokked the logic of storing key code and flags in the `Key` class correctly, that would not be possible without functionality. Now I just need to find some way to store the reference.

Reply:
On second thought, simply using the sum of these values seems bound to go wrong. I might have to store both key values in PROGMEM (or EEPROM) and return the sum of `QK_FIRST` and the returned index.